var/spack/repos/builtin/packages/cryptopp/package.py

```diff
@@ -28,4 +28,4 @@ def url_for_version(self, version):
         version_tuple = tuple(v for v in iter(version))
         version_string = reduce(lambda vs, nv: vs + str(nv), version_tuple, "")
```

Comment:
This seems a little... excessive. I know you didn't write it, but while you're at it, why not just use something like `version.replace(".", "")`? You might need to cast it to a string first: `str(version).replace(".", "")`